src/panel/context/schema-transforms.tsx

```diff
@@ -0,0 +1,21 @@
+import { GraphQLSchema, extendSchema, parse } from "graphql";
+
+const populateSchema = parse(`
+  directive @populate on FIELD
+`);
+
+export const appendPopulateDirective = (
+  schema: GraphQLSchema
+): GraphQLSchema => {
+  try {
+    return extendSchema(schema, populateSchema);
+  } catch (err) {
+    if (
+      err.message.startsWith(
```

Comment:
If populate exists already `extendSchema` will throw. In this case we want to return the schema as is